docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
+    inet 10.102.1.11/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+   $ IP_ADDRESS=10.102.1.11
+   ```
+
+   On CSM versions prior to 1.2, the CAN IP can be determined by querying the IP address of the `vlan007` interface.
+
+   ```
+   $ ssh ncn-m001 ip addr show vlan007
+   13: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+    inet 10.102.1.10/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+   $ IP_ADDRESS=10.102.1.10
+   ```
+
+   Once the IP address is determined, add an entry to `/etc/hosts` mapping the IP address to the hostname `kubernetes`.
+
+    ```
+    $ echo "${IP_ADDRESS} kubernetes" | sudo tee -a /etc/hosts
+    10.102.1.11 kubernetes
+    ```
+
+1. Modify `~/.kube/config` to set the cluster server address.
+
+   The value of the `server` key for the `kubernetes` cluster under the `clusters` section should be set to
+   `https://kubernetes:6443`.
+
+    ```yaml
+    ---
+    clusters:
+    - cluster:
+        certificate-authority-data: REDACTED
+        server: https://kubernetes:6443
+      name: kubernetes
+    ...
+    ```
+
+1. Confirm that `kubectl` can access the CSM Kubernetes cluster.
+
+    ```
+    $ kubectl get nodes
+    NAME       STATUS   ROLES    AGE    VERSION
+    ncn-m001   Ready    master   135d   v1.19.9
+    ncn-m002   Ready    master   136d   v1.19.9
+    ncn-m003   Ready    master   136d   v1.19.9
+    ncn-w001   Ready    <none>   136d   v1.19.9
+    ncn-w002   Ready    <none>   136d   v1.19.9
+    ncn-w003   Ready    <none>   136d   v1.19.9
+    ```
+
+1. Use `sat init` to create a configuration file for SAT.
+
+    ```
+    $ sat init -o /home/user/.config/sat/sat.toml
+    INFO: Configuration file "/home/user/.config/sat/sat.toml" generated.
+    ```
+
+1. Edit the SAT configuration file to set the API and S3 hostnames.
+
+   Externally available API endpoints are given domain names in PowerDNS, so the endpoints in the configuration file
+   should each be set to `subdomain.system-name.site-domain`, where `system-name` and `site-domain` are replaced with
+   the values specified during `csi config init`, and `subdomain` is the DNS name for the externally available service.
+   More information can be found under the [Externally Exposed
+   Services](https://cray-hpe.github.io/docs-csm/en-12/operations/network/customer_accessible_networks/externally_exposed_services/)
+   page in the CSM documentation.
+
+   The API gateway has the subdomain `api`, and S3 has the subdomain `s3`. The S3 endpoint runs on port 8080. The
+   following options should be set in the SAT configuration file:
+
+    ```toml
+    ...
```

Comment:
These get highlighted in red in the rendered docs, probably because they're not valid toml. Should we just leave them out in these snippets?

Reply:
I wanted to indicate that there are other lines in the configuration that should not be edited. However, looking at the other parts of the install instructions, such a convention isn't already in use, and we are just putting snippets in verbatim. I don't feel to strongly, so I will just remove the ellipses.

Reply:
Yeah, I do like to use the ellipsis convention myself, but it just renders a bit funny. Maybe removing the `toml` syntax highlighting would get rid of that. I noticed it didn't happen for yaml syntax.